exercises/starknet/README.md

```diff
@@ -1,7 +1,87 @@
 # Starknet Smart Contracts
 
-Cairo is the language used to develop Smart Contracts on Starknet.
+A Starknet contract is pretty much a Cairo module, or rather, it's represented by one.
```

Comment:
> A Starknet contract is pretty much a Cairo module, or rather, it's represented by one.

A Starknet contract can be created by annotating a Cairo module with the `#[contract]` attribute